WoWPro_Leveling/Wrath/Neutral/WOTLK_ZulDrak.lua

```diff
@@ -68,62 +68,62 @@ C Dressing Down|QID|12648|N|Return to the Reliquary of Agony and use the choker.
-A The Storm King's Vengeance|QID|12919|PRE|12916|N|From Gymer.|
-N The Storm King's Vengeance|QID|12919|N|Do not bother to look for the scourge: they will find you.|
-N The Storm King's Vengeance|QID|12919|N|Talk to Gymer to mount him and go!|
-C The Storm King's Vengeance|QID|12919|QO|1|N|Finish killing scourge.|S|
+A The Storm King's Vengeance|QID|12919|PRE|12916|N|From Gymer.|M|19.7,56.4|
+C The Storm King's Vengeance|QID|12919|N|Talk to Gymer to mount him and go! (Option 3)|M|19.7,56.4|S!US|QO|1|
 K Algar the Chosen|QID|12919|QO|2|M|29.2,46|N|He flies in circles underneath Volatrus, above the circular platform on the ground.|
 K Prince Navarius|QID|12919|QO|4|M|29.8,65.4;32.4,63.6|CN|N|At one of these two spots.|
-K Thrym|QID|12919|QO|3|N|He will spawn after killing the Prince. Snack on clouds and wait.|
-C The Storm King's Vengeance|QID|12919|QO|1|N|Finish killing scourge.|US|
+K Thrym|QID|12919|QO|3|N|He will spawn after killing the Prince. Snack on clouds and wait.|M|PLAYER|
 T The Storm King's Vengeance|QID|12919|N|Back to Crusader Forward Camp.|M|25.43,63.97|
-A Dark Horizon|QID|12664|O|N|You got this because you didn't finish the Drakuru quest chain in Grizzly Hills.|M|27.1,46.2|
-C Dark Horizon|QID|12664|O|N|Circle around the platform until you find a ramp on the outer ring. Gorebag is up there. Enjoy the ride.|M|29.76,47.90|
-T Dark Horizon|QID|12664|O|M|27.1,46.2|N|To Overlord Drakuru.|
-A Reunited|QID|12663|O|N|You got this because you finished the Drakuru quest chain in Grizzly Hills.|M|27.1,46.2|
-C Reunited|QID|12663|O|N|Circle around the platform until you find a ramp on the outer ring. Gorebag is up there. Enjoy the ride.|M|29.76,47.90|
-T Reunited|QID|12663|O|N|To Overlord Drakuru.|M|27.1,46.2|
-T Infiltrating Voltarus|QID|12661|N|Port back down, blow the horn.|U|41390|
+R Volatrus|ACTIVE|12661|M|28.16,44.84|N|Use the Choker and step onto the center of the pedestal|U|38699|CC|
+A Dark Horizon|PRE|-12238|QID|12664|N|You got this because you didn't finish the Drakuru quest chain in Grizzly Hills.|M|27.1,46.2|
+C Dark Horizon|QID|12664|N|Circle around the platform until you find a ramp on the outer ring. Gorebag is up there. Enjoy the ride.|M|29.76,47.90|
+T Dark Horizon|QID|12664|M|27.1,46.2|N|To Overlord Drakuru.|
+A Reunited|PRE|12238|QID|12663|N|You got this because you finished the Drakuru quest chain in Grizzly Hills.|M|27.1,46.2|
+C Reunited|QID|12663|N|Circle around the platform until you find a ramp on the outer ring. Gorebag is up there. Enjoy the ride.|M|29.76,47.90|
+T Reunited|QID|12663|N|To Overlord Drakuru.|M|27.1,46.2|
+T Infiltrating Voltarus|QID|12661|N|Port back down, blow the horn.|U|41390|M|28.25,45|
 A So Far, So Bad|QID|12669|PRE|12661|N|From Stefan Vadu.|
+C So Far, So Bad|ACTIVE|12669|QO|2|N|Use the Diluting Additive near the cauldrons. Blightguards will spawn mobs, run away to try to avoid aggro. The Cauldrons are at the waypoints.|U|39154|M|31.29,43.2;31.16,44.31;31.04,45.16;29.6,47.43;26.80,46.92|CS|
```

Comment:
C steps almost always should have their primary quest (the one being used for `QO`) in a QID tag not ACTIVE.